.github/workflows/verify-gdrive.yml

```diff
@@ -0,0 +1,21 @@
+name: "New Check Google Drive Access"
+on:
+  workflow_dispatch:
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    env:
+      REPO_OWNER: ${{ github.repository_owner}}
+      REPO_BRANCH: ${{ github.ref_name }}
+      SERVICE_ACCOUNT_KEY_JSON: ${{ secrets.SERVICE_ACCOUNT_KEY_JSON }}
```

Comment:
This SERVICE_ACCOUNT_KEY_JSON... this was what you downloaded directly from the Google Developers Console - right?

Reply:
yes.  It is downloaded as a JSON file - the key for one of the service accounts.  The contents of that file is placed in the SERVICE_ACCOUNT_KEY_JSON secret and passed down to python code where it is used to access Google Drive using the credentials of the service account.